src/resources/postcss/components/full/_ical-link.pcss

```diff
@@ -3,3 +3,45 @@
  * iCal Link
  *
  * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.tribe-events-c-ical__link {
+		align-items: center;
+		color: var(--color-accent-primary);
+		border: 1px solid var(--color-accent-primary);
+		border-radius: var(--border-radius-default);
+		display: inline-flex;
+		font-weight: var(--font-weight-bold);
+		padding: var(--spacer-2);
```

Comment:
this should be in skeleton.

Reply:
Thanks for the review!

I left the padding on the full stylesheet because it really did to the styles of the button, I thought it was more appropriate there.

1. I see all with the same blue.
2. Nice, `twentyseventeen` was having something with the hover.
3. Not sure if we're gonna use this in any other place for now. If we move it, would be cool to make it part of the design system in invision, and find a name, so it doesn't get lost.